test/pdfs/.gitignore

```diff
@@ -257,6 +257,7 @@
 !glyph_accent.pdf
 !personwithdog.pdf
 !find_all.pdf
+!issue13487.pdf
```

Comment:
This is still the 10 MB file, please don't use that one since it's way too large!

Reply:
Also, with a *linked* test-case there's no need for editing this file.